docs/UserGuide.md

```diff
@@ -209,22 +214,24 @@ Note that you need to have an account registered with HomeBoss._
 
 > Updates your user account details.
 
-Want to keep your account secure? You can update your account details by calling this command.
+_You can update your account details to keep your account secure by calling this command._
 
 **Format:** `update [--user USERNAME] [--password PASSWORD --confirmPass CONFIRM_PASSWORD]
-[--secretQn SECRET_QUESTION --answer ANSWER]`
+[--secretQn SECRET_QUESTION --answer ANSWER]`{.swift}
 
-<box background-color="#dff0d8" border-color="#d6e9c6" type="info" header="Note">
-    * [] around a group of parameters indicates that it is optional, but all parameters within the group must be provided.
-    * At least one of the optional fields must be provided.
-    * If PASSWORD is provided, CONFIRM_PASSWORD must also be provided, vice versa.
-    * If SECRET_QUESTION is provided, ANSWER must also be provided, vice versa.
-    * The details will be updated without checking against the current details.
+<box type="note" background-color="#dff0d8" border-color="#d6e9c6" icon=":information_source:">
+
+**Note:**
+* `[]`{.swift} around a group of parameters indicates that it is optional, but all parameters within the group must be provided.
```

Comment:
This would be mentioned in the getting started but now sure if it should be mentioned again. Any thoughts?

Reply:
Agree that it's repetitive, have removed it